integration/integration_mta_build_test.go

```diff
@@ -11,7 +11,7 @@ func TestMavenProject(t *testing.T) {
 	t.Parallel()
 	container := givenThisContainer(t, IntegrationTestDockerExecRunnerBundle{
 		Image:   "devxci/mbtci:latest",
-		User:    "mta",
+		User:    "root",
```

Comment:
Does it also have to be changed for when the mtaBuild step runs in any given pipeline (with the new MTA image)?